### ISSUE TITLE
Update TGM Plugin Activation generator to the latest version

### ISSUE THREAD

**Issue by dashkevych** [thomasgriffin/TGM-Plugin-Activation#782]
I've noticed that some changes were made to the `class-tgm-plugin-activation.php` file to address some Sniff issues: https://github.com/TGMPA/TGM-Plugin-Activation/commit/0a9b8c97c5ef0219d5863898a0519ac68caf1dee

I would be great if the generator (http://tgmpluginactivation.com/download/) used the latest version of TGM-Plugin-Activation

**Comment by jrfnl** [comment]
The Custom Download Generator uses the latest _released_ version of TGMPA.

I'd be happy to create a new release if we could get the issues listed here fixed, preferably via community PRs: https://github.com/TGMPA/TGM-Plugin-Activation/issues/741#issuecomment-488142682

**Comment by iltaen** [comment]
Custom Generator still has this bug, tried just now (fixed in this repo): 
https://github.com/TGMPA/TGM-Plugin-Activation/pull/670

**Comment by jrfnl** [comment]
@iltaen Yup, and my previous comment asking for people to help fix a few bugs before we release a new version still stands.
